voxel51/apps/api.py

```diff
@@ -228,6 +228,52 @@ def delete_analytic(self, analytic_id):
         res = self._requests.delete(endpoint, headers=self._header)
         _validate_response(res)
 
+    # DATA ####################################################################
+
+    def query_application_data(self, data_query):
+        '''Performs a customized data query at the application level (e.g.
+            includes all application users sans application admins).
```

Comment:
Please stop making a distinction between `application users` and `application admins` in documentation. App admins are administrators of the application and cannot upload data, run jobs, and do other user-y things.

The fact that our implementation once had a confusion between these things is not relevant to users of our application.

An example docstring might be:

```py
'''Performs a customized data query at the application level.

Queries using this route return results for all users of the application.
'''
```